sympy/printing/latex.py

```diff
@@ -780,12 +780,14 @@ def _print_UndefinedFunction(self, expr):
     @property
     def _special_function_classes(self):
         from sympy.functions.special.tensor_functions import KroneckerDelta
-        from sympy.functions.special.gamma_functions import gamma, lowergamma
+        from sympy.functions.special.gamma_functions import gamma,\
```

Comment:
The modifications in this file don't seem to be related to linear groups. Are then in this PR by accident?